generator/src/main.rs

```diff
@@ -81,12 +89,21 @@ fn get_table_data() -> (Vec<(char, String)>, Vec<(char, char)>) {
                 } else {
                     panic!("unknown hangul syllable {}", name)
                 }
+<<<<<<< HEAD
+            } else {
+=======
+            } else if !alias.is_empty() {
+>>>>>>> 6ea912b (Add alias search fallback for character())
```

Comment:
unresolved merge conflict

Reply:
my bad, I ran test again before final push this time.